src/main/resources/META-INF/rewrite/junit5.yml

```
@@ -266,3 +267,20 @@ recipeList:
   - org.openrewrite.java.testing.cleanup.AssertEqualsNullToAssertNull
   - org.openrewrite.java.testing.cleanup.AssertFalseNullToAssertNotNull
   - org.openrewrite.java.testing.cleanup.AssertionsArgumentOrder
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.testing.junit5.UseXMLUnitLegacy
+displayName: Use XMLUnit Legacy for JUnit 5
+description: Migrates XMLUnit 1.x to XMLUnit legacy 2.x
+tags:
+  - testing
+  - junit
+  - xmlunit
+recipeList:
+  - org.openrewrite.maven.ChangeDependencyGroupIdAndArtifactId:
+      oldGroupId: xmlunit
+      oldArtifactId: xmlunit
+      newGroupId: org.xmlunit
+      newArtifactId: xmlunit-legacy
+      newVersion: 2.9.1
```

Looking at https://github.com/xmlunit/xmlunit/releases it seems there's infrequent releases.
Would you think it makes sense to adopt [version selectors](https://docs.openrewrite.org/reference/dependency-version-selectors) here?
We could use for instance the following to get any future 2.9.x or 2.10+ release added:
```suggestion
      newVersion: 2.x
```